csharp/Valkey.Glide.IntegrationTests/Unit/GildeStringTests.cs

```diff
@@ -1,6 +1,6 @@
 ﻿// Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
 
-namespace Tests.Unit;
+namespace Valkey.Glide.IntegrationTests.Unit;
```

Comment:
Probably get rid of `IntegrationTests` in the middle?

On another side it is still IT, because it starts server

Reply:
Splattered into two projects